pages/profile.js

```diff
@@ -3,187 +3,165 @@ import Link from 'next/link'
-        containerStyle={'flex flex-wrap border-b border-gray-200'}
-        selectedTabStyle={
-          'bg-gray-100 active border-b-2 border-b-bright-blue-light'
-        }
-        unselectedTabStyle={
-          'text-gray-500 hover:text-gray-600 hover:bg-gray-50'
-        }
-        genericTabStyle={'inline-block text-lg py-3 px-4 text-center'}
-        locale={props.locale}
-      />
-      {tabSelected === 0 ? (
-        <>
-          <TabList
-            titles={[t.ei, t.cpp, t.oas]}
-            onClick={(index) => setProfileTabSelected(index)}
-            tabSelected={profileTabSelected}
-            containerStyle={'my-4 flex flex-wrap gap-4'}
-            selectedTabStyle={'bg-bright-blue-lighter'}
-            unselectedTabStyle={'bg-gray-lighter'}
-            genericTabStyle={
-              'text-center px-10 py-1 whitespace-nowrap rounded-xl hover:bg-bright-blue-lighter '
-            }
-            locale={props.locale}
-          ></TabList>
-          {profileTabSelected === 0 || !profileTabSelected ? (
-            <ProfileInfo
-              fields={[fakeFieldsEI, fakeFieldsEI2]}
-              locale={props.locale}
-            />
-          ) : profileTabSelected === 1 ? (
-            <ProfileInfo fields={[fakeFieldsCPP]} locale={props.locale} />
-          ) : profileTabSelected === 2 ? (
-            <ProfileInfo fields={[fakeFieldsOAS]} locale={props.locale} />
-          ) : null}
-        </>
-      ) : tabSelected === 1 ? (
-        <ProfileInfo fields={[fakeSecurityFields]} locale={props.locale} />
-      ) : null}
+      <Heading title={t.profileSettings} />
+      <span className="text-lg pb-5 block">{t.updateProfile}</span>
+
+      {benefitInformations.map((benefitInfo, index) => {
+        return (
+          <>
+            <h2 className="text-3xl pt-12 font-bold">{benefitInfo.benefit}</h2>
+            <div className="pb-16 pt-2 border-b border-gray-500">
+              {benefitInfo.show ? (
+                <ProfileInfo
+                  fields={benefitInfo.fields}
+                  locale={props.locale}
+                  editText={t.edit}
+                />
+              ) : null}
+              <ViewMoreLessButton
+                expanded={benefitInfo.show}
+                icon={benefitInfo.show}
+                onClick={() => benefitInfo.setShow(!benefitInfo.show)}
+                caption={!benefitInfo.show ? t.viewSettings : t.viewLess}
+                id={'view-more-less-' + index}
+                className="pt-6"
+                iconStyle="pl-0 pr-3"
+              />
+            </div>
+          </>
+        )
+      })}
+
+      <div className="mt-10">
+        <h2 className="text-3xl font-bold">Looking for security settings?</h2>
+        <ul className="list-disc ml-8 text-lg">
+          <Link href={'/security-settings'} passHref>
+            <li>
+              <a className="underline text-blue-600 cursor-pointer hover:text-blue-800 visited:text-purple-600">
```

Comment:
Just one last thing, we should use the colours from our pallet and not tailwinds.
```suggestion
              <a className="underline text-blue-default cursor-pointer hover:text-blue-hover visited:text-purple-medium">
```

Reply:
Done. Should I also change the `border-gray-500` for the line separators? The closest colour in our palette (gray-dark) is fairly different compared to the actual design.

Reply:
yes... you could use the HorizontalRule component which we have in other places, it's using border-gray-light
There's also a design system component we aren't using that does the same thing.